bridges/ARDMediathekBridge.php

```diff
@@ -0,0 +1,41 @@
+<?php
+class ARDMediathekBridge extends BridgeAbstract {
+	const NAME = 'ARD-Mediathek Bridge';
+	const URI = 'https://www.ardmediathek.de';
+	const DESCRIPTION = 'Feed of any series in the ARD-Mediathek, specified by its URI';
+	const MAINTAINER = 'yue_dongchen';
+
+	const PARAMETERS = array(
+		array(
+			'uri' => array(
+				'name' => 'URI',
+				'required' => true,
+				'defaultValue' => '45-min/Y3JpZDovL25kci5kZS8xMzkx/'
```

Comment:
"45-min/Y3JpZDovL25kci5kZS8xMzkx/" is not an URI. But if you don't have better naming, I will accept this.

Reply:
"Path" would perhaps be a better name. Also, that trailing `/` is maybe redundant; let's remove it.

Reply:
Do you know if `getSimpleHTMLDOM` would load redirects?